.github/workflows/ci.yml

```diff
@@ -132,6 +132,35 @@ jobs:
             ssl: no_ssl
             debug: debug
             hup: no_hup
+          # macOS - Wasmer (full)
+          - runtime: wasmer
+            wasmer: 3.1.1
+            os: macos-latest
+            cc: clang
+            ngx: 1.25.4
+            ssl: ssl
+            debug: debug
+            hup: no_hup
+          # macOS - Wasmtime
+          - runtime: wasmtime
+            wasmtime: 19.0.0
+            os: macos-latest
+            cc: clang
+            ngx: 1.25.4
+            ssl: ssl
+            debug: debug
+            hup: hup
+            path: t/01-wasm
+          # macOS - V8
+          - runtime: v8
+            v8: 12.0.267.17
+            os: macos-latest
+            cc: clang
+            ngx: 1.25.4
+            ssl: ssl
+            debug: debug
+            hup: no_hup
+            path: t/01-wasm
```

Comment:
I think this also needs the `carryforward` update for coverage.

Reply:
Good catch, thx!